core/js/typespecs/api_types.ts

```diff
@@ -391,6 +391,10 @@ export const datasetEventSchema = z
   .openapi("DatasetEvent");
 export type DatasetEvent = z.infer<typeof datasetEventSchema>;
 
+const functionTypeEnumNullish = functionTypeEnum
```

Comment:
Just to preface, none of the schema modifications are actually changing the "logical meaning" of the schema. They are just adding metadata / other workarounds for the sake of better conversion to an openAPI spec. So from a review perspective, just spot-checking that the schema changes are all behavior-preserving should be sufficient.